app/Http/Controllers/Api/TagsController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rogue\Http\Controllers\Api;
+
+use Rogue\Http\Requests\TagsRequest;
+use Rogue\Repositories\PostRepository;
+use Rogue\Http\Transformers\PostTransformer;
+
+class TagsController extends ApiController
+{
+    /**
+     * The post service instance.
+     *
+     * @var Rogue\Repositories\PostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \League\Fractal\TransformerAbstract;
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+    }
+
+    public function store(TagsRequest $request)
```

Comment:
@DFurnes Is this still the code we want to use for both controllers though? All the work I did on the front end was a result of switching to use this code in the `Controllers/TagsController`. Since it's happy with this code now, it's easy enough to move it out of the trait and back into both controllers. I find that to be less DRY but also easier to follow, so I could go either way